src/Microsoft.VisualStudio.Debugger.Interop.DAP/Microsoft.VisualStudio.Debugger.Interop.DAP.cs

```diff
@@ -51,6 +51,15 @@ public interface IDebugProgramDAP
         /// <param name="pResult">Number of bits in a pointer.</param>
         /// [PreserveSig]
         int GetPointerSize([Out] out int pResult);
+
+        /// <summary>
```

Comment:
I would suggest putting this is another interface since an engine might want to be hosted by OpenDebugAD7, but not support this feature.